package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "dt-app",
+  "version": "0.0.0",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test",
+    "lint": "ng lint",
+    "e2e": "ng e2e"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~8.2.9",
+    "@angular/cdk": "~7.3.7",
```

Comment:
Dlaczego wersja ~7.3.7 zamiast ~8.2.3? Angular 8 to Material też powinien być dla 8 :)